clients/ui/manifests/user-rbac/kubeflow-dashboard-rbac.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
A couple of things for this:

1. `admin-rbac.yaml` should be a base file now that RBAC by serviceaccount is mandatory.
2. So `user-rbac` should only have `kubeflo-dashboard-rbac.yaml` (I would rename it to something like `default-user-role.yaml` or something like that.
3. If you wanna be pro-kustomize, just change `kustomization.yaml`, adding `- ../base` as resource, so this is a proper kustomize overlay
4. If so, change the deploy script to only deploy the overlay and everything will be deployed.